cdap-standalone/src/main/java/co/cask/cdap/StandaloneMain.java

```diff
@@ -274,8 +277,12 @@ public static void main(String[] args) {
       }
       main.startUp();
     } catch (Throwable e) {
-      System.err.println("Failed to start Standalone CDAP. " + e.getMessage());
-      LOG.error("Failed to start Standalone CDAP", e);
+      Throwable rootCause = Throwables.getRootCause(e);
+      if (rootCause instanceof ServiceBindFailure) {
+        LOG.error("Failed to start Standalone CDAP: {}", rootCause.getMessage());
```

Comment:
should still keep the System.err.println, otherwise the user won't see anything on the command line when this happens